packages/php-wasm/logger/src/test/logger.spec.ts

```diff
@@ -1,29 +1,11 @@
-import { NodePHP } from '@php-wasm/node';
-import { LatestSupportedPHPVersion } from '@php-wasm/universal';
-import { Logger, addCrashListener } from '../lib/logger';
-import { collectPhpLogs } from '../lib/log-collector';
+import { Logger } from '../lib/logger';
 import { clearMemoryLogs, logToMemory } from '../lib/log-handlers';
 
 describe('Logger', () => {
-	let php: NodePHP;
 	const logger = new Logger([logToMemory]);
 	beforeEach(async () => {
-		php = await NodePHP.load(LatestSupportedPHPVersion);
 		clearMemoryLogs();
 	});
-	it('Event listener should work', () => {
-		const listener = vi.fn();
-		collectPhpLogs(logger, php);
-		addCrashListener(logger, listener);
-		php.dispatchEvent({
-			type: 'request.error',
-			error: new Error('test'),
-		});
-		expect(listener).toBeCalledTimes(1);
-
-		const logs = logger.getLogs();
-		expect(logs.length).toBe(1);
-	});
```

Comment:
This test would require a lot of mocking and it's not that useful, so I removed it.